frontend/docs/pages/home/features/retries/simple.mdx

```diff
@@ -19,15 +19,37 @@ This simple retry mechanism can help to mitigate transient failures, such as net
 
 To enable retries for a step in your workflow, simply add the `retries` property to the step object in your workflow definition:
 
+<Tabs items={['Python', 'Typescript', 'Go']}>
+  <Tabs.Tab>
+```python
+@hatchet.step(timeout='30s', retries=3)
+def step1(self, context: Context):
+    pass
+```
+  </Tabs.Tab>
+  <Tabs.Tab>
 ```typescript
 import { CreateStepSchema } from "@hatchet-dev/typescript-sdk";
 
-const myStep: z.infer<typeof CreateStepSchema> = {
-  name: "my-step",
-  timeout: "30s",
-  retries: 3,
+const step1: z.infer<typeof CreateStepSchema> = {
+name: "step1",
+timeout: "30s",
+retries: 3,
 };
-```
+
+````
+  </Tabs.Tab>
+  <Tabs.Tab>
+```go
+Steps: []*worker.WorkflowStep{
+  worker.Fn(func(ctx worker.HatchetContext) (result *stepOneOutput, err error) {
+    continue
+  }).SetName("step1").SetRetries(3).SetTimeout("30s"),
+},
+````
```

Comment:
Pretty sure the four backticks are unnecessary if you leave a newline in between the `<Tabs.Tab>`

Reply:
Some kind of linter issue, fixing in the next commit